packages/vscode-extension/package.nls.json

```diff
@@ -157,6 +157,30 @@
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorTitle": "Manage Collaborator",
     "teamstoolkit.commandsTreeViewProvider.manageCollaboratorDescription": "Manage M365 Teams App (with Microsoft Entra app) Collaborators",
     "teamstoolkit.commandsTreeViewProvider.addWebpartTitle": "Add SPFx Web Part",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployTitle": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.officeDevDeployDescription": "Deploy",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceTitle": "Publish",
+    "teamstoolkit.commandsTreeViewProvider.publishAppSourceDescription": "Link to the wiki about how to publish the add-in to AppSource",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInTitle": "Create a New Add-in",
+    "teamstoolkit.commandsTreeViewProvider.createOfficeAddInDescription": "Create a new add-in project of Word Excel or Powerpoint",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesTitle": "Check and Install Dependencies",
+    "teamstoolkit.commandsTreeViewProvider.checkAndInstallDependenciesDescription": "Check and install dependencies",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugTitle": "Preview Your Add-in (F5)",
+    "teamstoolkit.commandsTreeViewProvider.officeDevLocalDebugDescription": "Local debug your Add-in App",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestTitle": "Validate Manifest",
+    "teamstoolkit.commandsTreeViewProvider.validateManifestDescription": "Validate the manifest of Office add-ins project",
+    "teamstoolkit.commandsTreeViewProvider.editManifestTitle": "Edit Manifest",
+    "teamstoolkit.commandsTreeViewProvider.editManifestDescription": "Edit the manifest of Office add-ins project",
+    "teamstoolkit.commandsTreeViewProvider.generateManifestGUIDTitle": "Generate Manifest GUID",
+    "teamstoolkit.commandsTreeViewProvider.generateManifestGUIDDescription": "Generate new manifest GUID",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.officePartnerCenterTitle": "Open Partner Center",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.officePartnerCenterDescription": "Open Partner Center",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.getStartedTitle": "Get Started",
+    "teamstoolkit.commandsTreeViewProvider.officeAddIn.getStartedDescription": "Get Started about how to create Office Add-in project",
```

Comment:
Query: What do we want to convey here? Does this mean? - Get started to create Office add-in project. 
The current line is confusing.

Reply:
@supkasar the teamstoolkit.commandsTreeViewProvider.officeAddIn.getStartedTitle is the tree item title of "Get Started", which shows on UI as below. It only opens the url link currently. In the future it may open the walk through but now we haven't support the welcome walkthrough. So it only open the url link.
![image](https://github.com/OfficeDev/TeamsFx/assets/5762929/6d7aa2a0-6d2e-44c0-93c1-fd62bffff0d8)

It's different from the "Create a New Add-in" which is to create a real project.

Reply:
Thanks for the clarification. The description in L179 is little confusing - Get started about how to create Office Add-in project. 
Either it should be - Get started to create... or Learn more about how to...

Reply:
@supkasar Thanks for your suggestion. I've changed it to Learn more about how to... Would you please help review the PR again?

Reply:
@supkasar Sorry that the original PR is closed by mistake and it couldn't be reopened currently. Would you please help review the new one? https://github.com/OfficeDev/TeamsFx/pull/11004